tools/database/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "near-database"
```

Comment:
let's make it `near-database-tool` since `near-database` can be confused to the actual database implementation when considered in the global scope